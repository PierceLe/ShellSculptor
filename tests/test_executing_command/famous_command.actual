>> whoami
piercele
>> wc ~/test_data/movies.txt
wc: /Users/piercele/test_data/movies.txt: open: No such file or directory

