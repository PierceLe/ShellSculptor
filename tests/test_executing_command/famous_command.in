whoami
wc ~/test_data/movies.txt
