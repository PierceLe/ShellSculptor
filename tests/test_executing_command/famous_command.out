>> whoami
user
>> wc ~/test_data/movies.txt
  8  32 169 /home/test_data/movies.txt

